.gitignore

```diff
@@ -108,3 +108,10 @@ java/.mvn/.develocity/
 # rat
 filtered_rat.txt
 rat.txt
+
+/a
+/cpp/build_test/
+/cpp/src/arrow/util/config.h
+/cpp/src/parquet/parquet_version.h
+/pyarrow-dev/
+/python/myenv/
```

Comment:
Can we not include such changes unless absolutely necessary?

Reply:
My mistake i will correct it